manual/guides/blue_waters.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: manual
+title: Blue Waters
+subtitle: "Blue Waters - From Access to Utilization"
+permalink: /manual/guides/bluewaters
+---
+
+[Read this page first](https://bluewaters.ncsa.illinois.edu/getting-started)
+
+## Step 1. Communication
+Have Prof Huff get you approved for Blue Waters!
+
+## Step 2. Apply for Blue Waters
+Following the link and fill out the application
+	You can either receive a virtual token or a physical token
+	Virutal token is done through your mobile device,
+	and a physical token looks like this:
+ 
+![alt text](/img/manual/guides/bw-token.jpg)
+
+## Step 3. Wait
+Wait patiently for the token to arrive.
```

Comment:
Or, if you chose a soft token, install the RSA SecurID app and follow the instructions sent to you by email.